.github/workflows/build-docker.yml

```
@@ -31,7 +35,7 @@ jobs:
       - name: Build and push
         run: |
           dockertag=$(date +%Y%m%d).${{ github.run_number }}
-          if [[ "${{ github.ref }}" == "refs/heads/main" ]]; then
+          if ${{ env.IS_RELEASE }}; then
```

I think this only works with:
```suggestion
          if [[ "${{ env.IS_RELEASE }}" == "true" ]]; then
```